common.mk

```diff
@@ -2,7 +2,7 @@
 
 CXX=$(CHARM_HOME)/bin/charmc -language converse++
 
-OPTS?=-g3
+OPTS?=-g -O0 -Wall -Wextra
```

Comment:
Probably fine to include the `-W` flags, but `-O0` is a bit too extreme for CharmLite! Favor `-Og` at worst, maintain `-O2` for the general case, and `-Ofast` for benchmarks :)

Reply:
That was mostly to limit any optimization from the compiler's end while I debugged. I'll change that back to `-g3` in the next commit.

Reply:
Ah, note, `-Og` is generally optimal for debugging. Per the GCC manual, "some compiler passes that collect debug information are disabled at `-O0`."